app/src/main/java/com/framgia/oleo/screen/login/LoginFragment.kt

```diff
@@ -42,8 +52,11 @@ class LoginFragment : BaseFragment(), View.OnClickListener {
     override fun setUpView() {
         callBackManager = CallbackManager.Factory.create()
         binding.textViewLoginGG.setOnClickListener(this)
+        binding.buttonLogin.setOnClickListener(this)
```

Comment:
edit và chuyển nó xuống dưới cùng đồng loại đi e
```suggestion
       buttonLogin.setOnClickListener(this)
```